lib/web/views/help.haml

```diff
@@ -92,6 +94,13 @@
       %big
         %a#bookmarklet-download(onclick="return false")
           %u Narou.rbでダウンロード
+    %p
+      %span.glyphicon.glyphicon-bookmark
+      %big
+        %a#bookmarklet-download-and-mail(onclick="return false")
```

Comment:
__[HAML-Lint]__

Prefer the hash attributes syntax (%tag{ lang: 'en' }) over HTML attributes syntax (%tag(lang=en))


<!-- Comment by [SideCI](https://sideci.com) -->